diff -r 21b063d75b3e make/Main.gmk
--- jdk9/make/Main.gmk	Thu Mar 16 16:34:33 2017 +0000
+++ jdk9/make/Main.gmk	Mon Apr 03 19:55:39 2017 +0100
@@ -320,7 +320,7 @@
         ifneq ($(COMPILE_TYPE), cross)
 	  $(call LogWarn, Boot cycle build step 2: Building a new JDK image using previously built image)
 	  +$(MAKE) $(MAKE_ARGS) -f $(SRC_ROOT)/make/Init.gmk PARALLEL_TARGETS=$(BOOTCYCLE_TARGET) \
-	      JOBS= SPEC=$(dir $(SPEC))bootcycle-spec.gmk main
+	      JOBS=$(JOBS) SPEC=$(dir $(SPEC))bootcycle-spec.gmk main
         else
 	  $(call LogWarn, Boot cycle build disabled when cross compiling)
         endif
