diff -r b515beb3b4ad jdk9/hotspot/src/share/vm/utilities/globalDefinitions.hpp
--- jdk9/hotspot/src/share/vm/utilities/globalDefinitions.hpp	Thu Jul 07 18:40:53 2016 +0100
+++ jdk9/hotspot/src/share/vm/utilities/globalDefinitions.hpp	Tue Jul 12 19:13:51 2016 +0100
@@ -1163,8 +1163,10 @@
 #undef min
 #endif
 
+#ifndef _GLIBCXX_STDLIB_H
 #define max(a,b) Do_not_use_max_use_MAX2_instead
 #define min(a,b) Do_not_use_min_use_MIN2_instead
+#endif
 
 // It is necessary to use templates here. Having normal overloaded
 // functions does not work because it is necessary to provide both 32-
